pages/reviewing-code-for-cross-site-request-forgery-issues.md

```diff
@@ -21,13 +21,13 @@ can compromise the entire web application.
 
 ### Description of CSRF Vulnerabilities
 
-See the OWASP article on [CSRF](CSRF "wikilink") Vulnerabilities.
+See the OWASP article on [CSRF](https://owasp.org/www-community/attacks/csrf) Vulnerabilities.
 
 ### How to Test for CSRF Vulnerabilities
 
 See the [OWASP Testing
 Guide](:Category:OWASP_Testing_Project "wikilink") article on how to
-[Test for CSRF](Testing_for_CSRF_\(OWASP-SM-005\) "wikilink")
+[Test for CSRF](https://owasp.org/www-project-web-security-testing-guide/latest/4-Web_Application_Security_Testing/06-Session_Management_Testing/05-Testing_for_Cross_Site_Request_Forgery.html)

```

Comment:
This should adhere to: https://owasp.org/www-project-web-security-testing-guide/#how-to-reference-wstg-scenarios